packages/compiler/core/scanner.ts

```diff
@@ -320,6 +320,7 @@ export enum TokenFlags {
   Unterminated = 1 << 2,
   NonAscii = 1 << 3,
   DocComment = 1 << 4,
+  Backticked = 1 << 5,
```

Comment:
I don't think we want this flag. The token value should include the sigils around it, the parser should remove when necessary. Highlighting a token e.g. should I think highlight the entire token not just the contents.

Reply:
I think this is OK. We have a distinction between getTokenValue() and getTokenText(). The latter always returns the exact token text, and isn't influenced by this flag. Neither are the token start and end positions that getTokenText() uses. This is equivalent to how  getTokenValue() on a string literal token doesn't return the surrounding quotes and is unescaped. Highlighting is working correctly as far as I can tell in the playground.

Reply:
You're right, I forgot about the value and text distinction. Been too long since I've been in the scanner! A sadness.